### PR TITLE
fix: [PIE-4016]: Jenkins/step changes

### PR DESCRIPTION
**Pull request by satyam-harness**
##### Jira Links:
https://harness.atlassian.net/browse/PIE-4016

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by satyam-harness** [comment]
dockerize

**Comment by satyam-harness** [comment]
dockerize

**Comment by satyam-harness** [comment]
retrigger typecheck

**Comment by satyam-harness** [comment]
retrigger typecheck

**Comment by satyam-harness** [comment]
trigger cypress

**Review by devinder-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepInputSet.tsx [hunk 1]
```diff
@@ -64,18 +64,24 @@ function JenkinsStepInputSet(formContentProps: any): JSX.Element {
...
 
-  const { refetch: refetchJobParameters } = useGetJobParametersForJenkins({
-    lazy: true,
-    jobName: ''
-  })
+  // To be uncommented when we support runtime jobParameters
+  // const { refetch: refetchJobParameters, data: jobParameterResponse } = useGetJobParametersForJenkins({
+  //   lazy: true,
+  //   jobName: ''
+  // })
```
Comment (devinder-harness) [inline src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepInputSet.tsx:84]: can we remove this `commented` code ?

**Review by devinder-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepInputSet.tsx (hunk 1 above)
Comment (satyam-harness) [reply src/modules/70-pipeline/components/PipelineSteps/Steps/JenkinsStep/JenkinsStepInputSet.tsx:84]: actually we will add support of jobParameter in few days, at that time we will need to uncomment few snippets, this is one of them, shall i keep it or remove it?

**Review by Sagar-Rajput7 — DISMISSED** [review]

**Review by shamala-harness — DISMISSED** [review]

**Review by PratyushGar — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**